charts/opentelemetry-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.100.0
+    tag: 0.100.1
```

Comment:
it seems like this actually got misaligned and we should be bumping this to match the operator version.